test/interfaces/sampler_interface.jl

```diff
@@ -0,0 +1,116 @@
+using QEDprocesses
+using QEDevents
+using QEDbase
+import Random: AbstractRNG, MersenneTwister
+using Distributions
+import Distributions: rand, rand!, _rand!
+
+RNG = MersenneTwister(708583836976)
+
+ATOL = 0.0
```

Comment:
atol is zero by default anyways and should pretty much never be not 0 unless we're expecting a zero in what we're `isapprox`ing, I think this can go.

Reply:
Correct, indeed. However, this explicit setting of `atol` makes your point independent from the default behavior of `isapprox`, which could change in future releases of Julia.

Reply:
That's fair enough, though that would be a very strange decision ;)